### ISSUE TITLE
Cannot find module 'jest-cli'

### ISSUE THREAD

**Issue by fetters5** [meltedspark/angular-builders#325]
**Describe the bug**
When creating a new angular 8 project and adding    
"@angular-builders/jest": "^8.0.1" 
"jest": "^24.8.0

 and trying to run a test I get the following error

```
Cannot find module 'jest-cli'
Error: Cannot find module 'jest-cli'
```

**To Reproduce**
1. create a new angular 8 project 
2. Add the jest  builder/dependency
3. update angular.json to the following
```
 "test": {
          "builder": "@angular-builders/jest:run",
          "options": {

          }
        },
```
3. run ng test

**Expected behavior**
I expect it to work as it did in Angular 7.  

**Screenshots**

**Builder:**
 - The name of the builder
 - The version of the builder

**Libraries**
`    "@angular-devkit/build-angular": "~0.800.0",`

**Additional context**
if I  npm install jest-cli as a package independently it works.  Do we need to install both jest and jest-cli as dev dependencies?

**Comment by meltedspark** [comment]
Actually `jest-cli` should be installed as part of `jest`. Please check:
1. What happens if you use Angular 7 and the exact same version of Jest
2. When using Angular 8 what's in `node_modules`? Is there a module `jest-cli`? Is it in `node_modules/jest/node_modules`? Is there a binary of Jest in `node_modules/.bin`?

**Comment by ogix** [comment]
I have the same problem. There is no `jest-cli` in `node_modules`. And there is a `jest` binary in `.bin` folder

**Comment by fetters5** [comment]
I have no issues with my project when it was on angular 7 using the latest version of jest 24

There is a jest binary in .bin, and there is a jest-cli folder inside of  `node_modules/jest/node_modules`

Below is the full error if that helps.  I am on Node v10.15.3. and this occurs on both my project and a fresh angular 8 project where i just added the jest builder

![image](https://user-images.githubusercontent.com/3276557/58634107-7335a680-82b8-11e9-8818-b229ac279bac.png)

**Comment by meltedspark** [comment]
OK, I'll look into it. Just to be clear, it's not a blocker, right? Installing `jest-cli` explicitly is a viable workaround?

**Comment by fetters5** [comment]
Correct, explicitly adding jest-cli as a workaround did the trick for me (at least in my initial testing).

Thanks for all your work on this project!

**Comment by jonathanlie** [comment]
Yep having the same issue now, +1 !
Will install jest-cli explicitly for now, watching for updates here.

**Comment by meltedspark** [comment]
Please confirm the issue is fixed in `8.0.2`

**Comment by jonathanlie** [comment]
Fixed for me. Thanks for the quick fix.
